build/actors/README.md

```diff
@@ -0,0 +1,18 @@
+# Bundles
+
+This directory includes the actors bundles for each release. Each actor bundle is a zstd compressed
+tarfile containing one bundle per network type. These tarfiles are subsequently embedded in the
+lotus binary.
+
+## Updating
+
+To update, run the `./pack.sh` script. For example, the following will pack the builtin actors release `dev/20220602` into the `v8` tarfile.
+
+```bash
+./pack.sh v8 dev/20220602
+```
+
+This will:
+
+1. Download the actors bundles and pack them into the appropriate tarfile.
+2. Run `make bundle-gen` in the top-level directory to regenerate the bundle metadata file.
```

Comment:
can one specify two versions at one time?

Reply:
No, you'd have to run the command for each version/release you want to download.

Reply:
Oh, I see. `make bundle-gen` will generate the metadata for all versions.